src/portaudio-sound.cpp

```diff
@@ -0,0 +1,276 @@
+/******************************************************************************\
+ * Copyright (c) 2021
+ *
```

Comment:
The Author is missing and has to be added. You can use your Github name if you like.